src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -24,7 +24,7 @@ public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
                 return;
             }
 
-            if (!options!.IOSNativeSupportEnabled)
+            if (!options!.IosNativeSupportEnabled)
```

Comment:
Event though "we know it's checked in Validate" it's hard to tell from reading here.
So not against having the `null` check in the extension method but it looks a bit magic or possibly looks like a bug from the outside. So better have a `null` check before the use of `options` here